server/member/member.go

```diff
@@ -173,7 +173,7 @@ func (m *Member) CampaignAndKeepLeader(ctx context.Context, leaseTTLSec int64, c
 		return ErrTxnPutLeader.WithCausef("txn put leader failed, resp:%v", resp)
 	}
 
-	m.logger.Info("succeed to set leader", zap.String("leader-key", m.leaderKey), zap.String("leader", m.Name))
+	m.logger.Info("[SetLeader]", zap.String("leader-key", m.leaderKey), zap.String("leader", m.Name))
```

Comment:
Is this a new log style? Maybe we should introduce it in the development guide docs if so.